### ISSUE TITLE
Seriously does not consume BUG

### ISSUE THREAD

**Issue by p82183645** [Zrips/CMI#1137]
Please fix cmi attachcommand
If use plugins Crazy Crates
Cmi attachcommand !limiteduse:1!;;!!cc!cc open testcrate [playerName]

Use the left button against the player
Will trigger the command (do not consume items <<< BUG

**Cmi Version (using`/cmi version`): 8.2.1.0

**Server Type (Spigot/Paperspigot/etc): paper

**Server Version (using `/ver`):** 1.12.2

**Relevant plugins (Delete if this isn't needed):** Crazy Crates , CMI

**Comment by LogGits** [comment]
Duplicate of #848 we are waiting on your reply there.
